packages/vue3/src/_dev/App.vue

```diff
@@ -1,63 +1,33 @@
 <script setup lang="ts">
 import 'uno.css';
 import '../assets/main.css';
-import { ref, watch } from 'vue';
-import { MediaInput, UploadedMedia } from '~/components';
+import { ref } from 'vue';
+import { RichText } from '~/components';
+import type { ToolbarButtonsType } from '~/components/RichText/types';
 
-const attachments = ref<File[]>([]);
-const disabled = ref(false);
-const limit = 4;
-
-const checkLimit = () => {
-  disabled.value = attachments.value.length >= limit;
-};
-
-const deleteFile = (file: File) => {
-  const idx = attachments.value.indexOf(file);
-  if (idx > -1) {
-    attachments.value.splice(idx, 1);
-    checkLimit();
-  }
+const content = ref('');
+const error = ref(false);
+const license = ref('license-key');
+const toolbar: ToolbarButtonsType = {
+  text: ['fullscreen', 'bold', 'italic', 'underline', 'strikeThrough', 'textColor', 'backgroundColor', 'fontSize', 'paragraphFormat'],
+  paragraph: ['align', 'formatOL', 'formatUL', 'outdent', 'indent'],
+  misc: ['undo', 'redo'],
+  rich: ['insertTable', 'insertImage', 'insertVideo', 'emoticons', 'insertLink', 'specialCharacters', 'insertHR', 'codeView', 'clear'],
 };
-watch(attachments, () => {
-  checkLimit();
-});
 </script>
 
 <template>
   <div class="container">
-    <div class="files-grid">
-      <UploadedMedia
-        v-for="(attachment, index) in attachments"
-        :key="index"
-        :file="attachment"
-        @delete="deleteFile(attachment)"
-      />
-    </div>
-    <MediaInput v-model="attachments" :limit="limit" :disabled="disabled" />
-    <div class="files-grid">
-      <UploadedMedia
-        file="https://cdn.pixabay.com/photo/2015/04/23/22/00/tree-736885_1280.jpg"
-      />
-    </div>
+    <RichText v-model="content" :error="error" lang="ar" :license="license" :toolbar="toolbar" />
   </div>
 </template>
 
-<style scoped>
+<style scoped lang="css">
```

Comment:
css is the default lang so we can omit this
```suggestion
<style scoped>
```